### PR TITLE
More updates to wip site update

### PR DESCRIPTION
**Pull request by lukewilliamboswell**
Spent some more time tinkering with the WIP site update. Still a lot more to do, but I wanted to get something that looks more acceptable. 

Screenshot 2023-08-02 at 17 01 27

### CODE REVIEW

**Comment by lukewilliamboswell** [comment]
@Anton-4 do you know why these links are being reported as broken? They definitely work when I run the site locally.

**Comment by Anton-4** [comment]
The link checker is not really set up to work with "local" links, I've set it up to ignore all .html links.

File: www/wip_new_website/content/community.md [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
+## Meet
 
 - Group Chat [roc.zulipchat.com](https://roc.zulipchat.com/)
 - Github Project [roc-lang/roc](https://github.com/roc-lang/roc)
 - Meetups & Events
 
-## Ideas & Proposals
+## Proposals
 
 - [Good First Issues](https://github.com/roc-lang/roc/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
```
Comment (Anton-4) [inline www/wip_new_website/content/community.md:21]: Does the "Good First Issues" link still make sense in this spot?

**Comment by Anton-4** [comment]
Nice improvements over the previous version @lukewilliamboswell!

I've made some changes here and there, I'll finish this up tomorrow.

File: www/wip_new_website/content/community.md (hunk 1 above)
Comment (lukewilliamboswell) [reply www/wip_new_website/content/community.md:21]: Oops, I didnt mean to remove that.

**Review by Anton-4 — APPROVED** [review]
Summary of my changes:
- Reduced amount of different font sizes. See the section "As Few Font Sizes as Possible" [here](https://www.learnui.design/blog/mobile-desktop-website-font-size-guidelines.html).
- Less text. I read this on builtformars.com once, but I can't find the exact page anymore. Anyway, nobody wishes they saw (almost) the same amount of information in more words :p Less text also makes searching/scanning easier and reduces cognitive load. 
- Changed footer to be less noticeable, only important things should stand out. 
- Reduced amount of different colors in color palette.
- Played with the colors a bit.

Thanks for working on this @lukewilliamboswell!

**Pull request merged**